include/modem/nrf_modem_lib.h

```diff
@@ -143,6 +143,19 @@ void nrf_modem_lib_shm_tx_diagnose(void);
  */
 void nrf_modem_lib_heap_diagnose(void);
 
+/**
+ * @brief Modem fault handler.
+ *
+ * Two default handlers are given in ``nrf_modem_lib.c`` for logging the fault (default) and
+ * reinitializing the modem. If other handling is required, the application can set
+ * ``CONFIG_NRF_MODEM_LIB_ON_FAULT_APPLICATION_SPECIFIC`` and implement the fault handler
+ * function: ``nrf_modem_fault_handler``.
```

Comment:
```suggestion
 * Two default handlers are given in @c nrf_modem_lib.c for logging the fault (default) and
 * reinitializing the modem. If other handling is required, the application can set
 * @c CONFIG_NRF_MODEM_LIB_ON_FAULT_APPLICATION_SPECIFIC and implement the fault handler
 * function: nrf_modem_fault_handler().
```

ref https://www.doxygen.nl/manual/commands.html

Reply:
Thanks, removed this part.